tests/test_data_sim.py

```diff
@@ -113,6 +125,8 @@ def test_getitem():
 def test_centers():
     sim_data = make_sim_data()
     for mon in sim_data.simulation.monitors:
+        if isinstance(mon, DiffractionEfficiencyMonitor):
```

Comment:
I think this is not need, basically nothing will happen unless `if isinstance(mon, (FieldMonitor, FieldTimeMonitor, ModeSolverMonitor))`

Reply:
The `DiffractionEfficiencyMonitor` inherits from `FieldMonitor` though, so that code snippet was being called and erroring out.

Reply:
ohh. got it. k, please disregard.